src/schemas/withdrawal.yaml

```diff
@@ -0,0 +1,21 @@
+Withdrawal:
+  type: object
+  title: Validator withdrawal
+  required:
+    - withdrawalIndex
```

Comment:
Should this just be `index` as per https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md#withdrawalv1 ?

Reply:
While it is not necessary that these two fields line up, it is probably a good idea for them to.

Reply:
yeah I like the verbosity but every other notation is just called `index` and there is value in minimizing surprise here